wrapper/src/main/java/software/amazon/jdbc/plugin/cache/CacheConnection.java

```diff
@@ -113,12 +114,26 @@ public class CacheConnection {
           null,
           "Explicit cache name for ElastiCache IAM authentication. ");
 
+  protected static final AwsWrapperProperty CACHE_CONNECTION_TIMEOUT =
+      new AwsWrapperProperty(
+          "cacheConnectionTimeOut",
+          null,
+          "Cache connection request timeout duration in milliseconds.");
+
+  protected static final AwsWrapperProperty CACHE_CONNECTION_POOL_SIZE =
+      new AwsWrapperProperty(
+          "cacheConnectionPoolSize",
+          null,
```

Comment:
Default should be "20", not `null`. 

Look at https://github.com/QuChen88/aws-advanced-jdbc-wrapper/blob/caching/wrapper/src/main/java/software/amazon/jdbc/plugin/cache/DataRemoteCachePlugin.java#L70 as an example.